app/src/main/java/com/epfl/drawyourpath/database/FireDatabase.kt

```diff
@@ -214,6 +215,41 @@ class FireDatabase : Database() {
         return updateUserData(dataUpdated)
     }
 
+    override fun addUserToFriendsList(userId: String): CompletableFuture<Boolean> {
+        val future = CompletableFuture<Boolean>()
+        isUserStoredInDatabase(userId).thenAccept {
+            if(!it){
+                future.completeExceptionally(java.lang.Error("The user with $userId is not present on the database."))
+            }else{
+                val newFriend = HashMap<String,Any?>()
+                newFriend.put(userId, null)
```

Comment:
Does it work? I think RDB removes keys with `null` values. We should simply store it as an array.

Reply:
I have done some research before doing this friendlist. And I think it s good to do like this. You can see an example on this link : https://stackoverflow.com/questions/45780290/structuring-a-friend-list-in-a-firebase-database